src/linux/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -200,6 +288,58 @@ AccessPointControllerLinux::SetProtocol(Microsoft::Net::Wifi::Ieee80211Protocol
     return isOk && m_hostapd.Reload();
 }
 
+bool
+AccessPointControllerLinux::SetAkmSuites(std::vector<Ieee80211AkmSuite> akmSuites)
+{
+    bool isOk = false;
+
+    // Construct the wpa_key_mgmt hostapd property value.
+    std::string concatenatedWpaKeyMgmtPropertyValue{};
+    for (const auto& akmSuite : akmSuites) {
+        const auto& wpaKeyMgmtPropertyValue = detail::IeeeAkmSuiteToWpaKeyMgmtPropertyValue(akmSuite);
+        concatenatedWpaKeyMgmtPropertyValue += wpaKeyMgmtPropertyValue + " ";    
+    }
+
+    try {
+        // Set the hostapd wpa_key_mgmt property.
+        isOk = m_hostapd.SetProperty(Wpa::ProtocolHostapd::PropertyNameWpaKeyMgmt, concatenatedWpaKeyMgmtPropertyValue);
+
+        // Set the hostapd wpa property.
+        isOk = isOk && m_hostapd.SetProperty(Wpa::ProtocolHostapd::PropertyNameWpa, Wpa::ProtocolHostapd::PropertyWpaValue2); // TODO: Set this correctly
+    } catch (const Wpa::HostapdException& ex) {
+        throw AccessPointControllerException(std::format("Failed to set AKM suites for interface {} ({})", GetInterfaceName(), ex.what()));
+    }
+
+    // Reload hostapd conf file.
+    return isOk && m_hostapd.Reload();
```

Comment:
For ease/clarity during debugging, can we separate the `Reload()` into it's own asssignment?

```suggestion
    // Reload hostapd conf file.
    isOk = isOk && m_hostapd.Reload();

    return isOk;
```

This way you can break on either line and see the current state (`isOk`) and the result of the `Reload` if `isOk` was `true`.